Code/Framework/AzCore/AzCore/Serialization/EditContext.h

```diff
@@ -235,6 +235,17 @@ namespace AZ
              */
             ClassBuilder*  ClassElement(Crc32 elementIdCrc, const char* description);
 
+
+             /**
+             * Declare element with attributes that belong to the class SerializeContext::Class, this is a logical structure, you can have one or more ClassElements.
+             * \uiId is the logical element ID (for instance "Group" when you want to group certain elements this class.
+             * then in each DataElement you can attach the appropriate group attribute.
```

Comment:
nit: This also needs a closing paren as it seems to be missing here or on the line above

Reply:
Thanks, will fix!